Live/LiveEventWithDVR/Program.cs

```diff
@@ -79,522 +74,375 @@
+
+// Optionally use Event Hub to monitor the Live Event
+if (!string.IsNullOrWhiteSpace(options.EVENT_HUB_NAME) && !options.EVENT_HUB_NAME.StartsWith("---"))
 {
-    #region CreateLiveEvent
-    Console.WriteLine($"Creating a live event named {liveEventName}");
-    Console.WriteLine();
+    mediaServicesMonitor = await MediaServicesMonitor.StartMonitoringAsync(
+        armClient,
+        mediaServicesResourceId,
+        liveEventName,
+        options,
+        credential);
+}
 
-    // Creating the LiveEvent - the primary object for live streaming in AMS. 
-    // See the overview - https://learn.microsoft.com/azure/media-services/latest/live-event-concept
+var liveEvent = await CreateLiveEventAsync(mediaServicesAccount, liveEventName);
 
-    // Create the LiveEvent
+var (liveOutput, asset) = await CreateLiveOutputAsync(mediaServicesAccount, liveEvent, assetName, liveOutputName, manifestName);
 
-    // Understand the concepts of what a live event and a live output is in AMS first!
-    // Read the following - https://learn.microsoft.com/azure/media-services/latest/live-events-outputs-concept
-    // 1) Understand the billing implications for the various states
-    // 2) Understand the different live event types, pass-through and encoding
-    // 3) Understand how to use long-running async operations 
-    // 4) Understand the available Standby mode and how it differs from the Running Mode. 
-    // 5) Understand the differences between a LiveOutput and the Asset that it records to.  They are two different concepts.
-    //    A live output can be considered as the "tape recorder" and the Asset is the tape that is inserted into it for recording.
-    // 6) Understand the advanced options such as low latency, and live transcription/captioning support. 
-    //    Live Transcription - https://learn.microsoft.com/azure/media-services/latest/live-event-live-transcription-how-to
-    //    Low Latency - https://learn.microsoft.com/azure/media-services/latest/live-event-latency-reference
+liveEvent = await StartLiveEventAsync(liveEvent);
 
-    // When broadcasting to a live event, please use one of the verified on-premises live streaming encoders.
-    // While operating this tutorial, it is recommended to start out using OBS Studio before moving to another encoder. 
+PrintIngestUrls(liveEvent);
 
-    // Note: When creating a LiveEvent, you can specify allowed IP addresses in one of the following formats:                 
-    //      IpV4 address with 4 numbers
-    //      CIDR address range  
+var streamingLocator = await DemoStreamingAsync(
+    mediaServicesAccount,
+    asset,
+    streamingEndpointName,
+    streamingLocatorName,
+    manifestName,
+    dvrAssetFilterName);
 
-    var allAllowIpRange = new IPRange()
-    {
-        Name = "AllowAll",
-        Address = IPAddress.Parse("0.0.0.0"),
-        SubnetPrefixLength = 0
-    };
+Console.WriteLine("Cleaning up resources, stopping Live Event billing, and deleting live Event...");
```

Comment:
casing on Live Event... be consistent everywhere.  Lower case or Upper case.

Reply:
Ahh... I thought I had fixed all the casing! Will update...